kalite/updates/management/commands/classes.py

```diff
@@ -1,31 +1,68 @@
 """
 """
 from datetime import datetime
+from optparse import make_option
 
 from django.conf import settings; logging = settings.LOG
 from django.utils.translation import ugettext as _
 
 from ...models import UpdateProgressLog
 from fle_utils.django_utils.command import LocaleAwareCommand
+from functools import wraps
+
+
+def skip_if_no_progress_log(func):
```

Comment:
This is necessary because progress logging depends on a database that doesn't exist when you're building a template! And since templates should not be dirty, we can't log to them anyways.